### PR TITLE
use W3C color names in MarlinPlotterInterface (for #587)

### PR DESCRIPTION
**Pull request by i-make-robots**

### CODE REVIEW

File: src/main/java/com/marginallyclever/convenience/W3CColorNames.java [hunk 1]
```diff
@@ -164,4 +169,16 @@ public class SVGColorNames {
     public static ColorRGB get(String name) {
         return map.get(name);
     }
+
+    /**
+     * @param match the color to matcch
+     * @return the name matching the given color, or null if no match.
+     */
+    public static String get(ColorRGB match) {
+        for(String key: map.keySet()) {
```
Comment (coliss86) [inline src/main/java/com/marginallyclever/convenience/W3CColorNames.java:178]: you don't have to iterate through the map, you can use : `map.get(match)`

Comment (coliss86) [reply src/main/java/com/marginallyclever/convenience/W3CColorNames.java:178]: And the method `isEqualsTo()` can be removed and the method `equals()` can be used in place.

Comment (i-make-robots) [reply src/main/java/com/marginallyclever/convenience/W3CColorNames.java:178]: There is only map.get(key), no map.get(value).  I am doing the latter.

isEqualsTo() is a weird method.  removed.  ty!

Comment (coliss86) [inline src/main/java/com/marginallyclever/convenience/W3CColorNames.java:178]: Have you considerd doing it with `stream` ? Here is an example : https://www.baeldung.com/java-map-key-from-value#a-functional-approach

Comment (i-make-robots) [reply src/main/java/com/marginallyclever/convenience/W3CColorNames.java:178]: Am I correct that stream would go through all items in the map?  My way it stops on the first hit.  slightly faster.

Comment (coliss86) [reply src/main/java/com/marginallyclever/convenience/W3CColorNames.java:178]: no, stream won't go to the end of the iteration, this is why they are built for.

**Pull request merged**